vpr/src/route/router_lookahead/router_lookahead.h

```diff
@@ -82,12 +82,6 @@ class RouterLookahead {
 /**
  * @brief Force creation of lookahead object.
  * @attention This may involve recomputing the lookahead, so only use if lookahead cache cannot be used.
```

Comment:
Not sure it is worth deleting @param just because it's not fully filled in (better to fill in the ones you know ...)